contracts/InsuranceFund.sol

```diff
@@ -60,6 +88,46 @@ contract InsuranceFund is IInsuranceFund, ReentrancyGuardUpgradeable, OwnerPausa
         emit Repaid(repaidAmount, tokenBalanceAfterRepaid);
     }
 
+    /// @inheritdoc IInsuranceFund
+    function distributeFee() external override nonReentrant whenNotPaused returns (uint256) {
+        address vault = _borrower;
+        address token = _token;
+        address surplusBeneficiary = _surplusBeneficiary;
+        int256 threshold = _threshold.toInt256();
+
+        // IF_SNS: surplusBeneficiary not yet set
+        require(surplusBeneficiary != address(0), "IF_SNS");
+
+        // IF_TEZ: threshold is equal to zero
+        require(threshold > 0, "IF_TEZ");
+
+        int256 insuranceFundFreeCollateral = IVault(vault).getFreeCollateralByToken(address(this), token).toInt256();
+
+        int256 insuranceFundCapacity = getInsuranceFundCapacity();
+
+        // We check IF's capacity against the threshold, which means if someone sends
+        // non-settlement collaterals to IF, it will increase IF's capacity and might help it meet the threshold.
```

Comment:
```suggestion
        // settlement collaterals to IF, it will increase IF's capacity and might help it meet the threshold.
```

This part seems incorrect.  Even if I deposit non-settlement collaterals to IF address, `getInsuranceFundCapacity ` won't get changed. Please correct me if I'm wrong.